docs/interface.rst

```diff
@@ -385,9 +385,32 @@ The follow event kinds are understood (see ``status.py`` for more details on the
 
 - ``"download-finished"``: same as upload version.
 
+- ``"invite-created"``: A new invite is created locally.
+All invite events also have a ``folder`` indicating which folder they pertain to, a ``id``, ``participant-name`` indicating the invitee and ``mode`` (``"read-only"`` or ``"read-write"``).
+
+- ``"invite-welcomed"``: Connection to the mailbox has succeeded. As well as the keys from ``"invite-created"`` the welcome data is contained in ``welcome`` (you should show users the ``"motd"`` if there is one).
+
+- ``"invite-code-created"``: We have allocated what we need on the mailbox server and have a valid code.
+As well as the keys from ``"invite-created"``, a ``code`` key will be included indicating the wormhole code.
+
+- ``"invite-received-versions"``: We have received an encrypted message from our peer.
```

Comment:
This is extreme bike-shedding (so feel free to ignore!) but "invite-versions-received" would be even more consistent with the other existing event names/kinds -- all of which end with with the past-tense verb (i.e., "-ed").